content/source/docs/cloud/integrations/service-now/service-catalog.html.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: "cloud"
+page_title: "Service Catalog - ServiceNow Service Catalog Integration - Terraform Cloud and Terraform Enterprise"
+description: |-
+  Configure Service Catalogs and VCS repositories to allow end users to
+  provision infrastructure using ServiceNow and Terraform Cloud.
+---
+
+# Service Catalog Configuration
+
+When using ServiceNow with the Terraform Cloud integration, you will configure
+at least one Service Catalog. You will also configure one or more Version
+Control System (VCS) repositories containing the Terraform configuration which
+will be used to provision that infrastructure. End users will request
+infrastructure from the Service Catalog, and Terraform Cloud will fulfill the
+request by creating a new Workspace, applying the configuration, and then
+reporting the results back to ServiceNow.
+
+## Prerequisites
+
+1. The Terraform Cloud integration software must be installed and configured on
+   your ServiceNow instance. These steps are covered in the [installation
+   documentation](./index.html).
+1. The The OAuth Token ID and repository identifier for each VCS repository that
+   Terraform Cloud will use to provision infrastructure. Your Terraform Admin
+   will provide these to you.
+1. Any Terraform or environment variables required by the configuration in the
+   given VCS repositories.
+
+Once these steps are complete, in order for end users to provision
+infrastructure with ServiceNow and Terraform Cloud, the ServiceNow Admin will
+perform the following steps to make Service Items available to your end users.
+
+1. Add at least one Service Catalog for use with Terraform.
+1. Configure at least one one VCS repository in ServiceNow.
+1. Create variable sets to define Terraform and environment variables to be used
+   by Terraform Cloud to provision infrastructure.
+
+## Add the Terraform Service Catalog
+
+-> **ServiceNow Role:** `admin`
+
+First, add a Service Catalog for use with the Terraform integration. Depending
+on your organization's needs, you might use a single service catalog, or
+several. If you already have a Service Catalog to use with Terraform, skip to
+the next step.
+
+1. In ServiceNow, open the Service Catalog > Catalogs view by searching for
+   "Service Catalog" in the left-hand navigation.
+   1. Click the plus sign in the top right.
+   1. Select "Catalogs > Terraform Catalog > Title and Image" and choose a
+      location to add the Service Catalog.
+   1. Close the "Sections" dialog box by clicking the "x" in the upper right-hand
+      corner.
+
+-> **Note:** In step 1, be sure to choose "Catalogs", not "Catalog" from the
+left-hand navigation.
+
+## Configure VCS Repositories
+
+-> **ServiceNow Roles:** `admin` or `x_terraform.vcs_repositories_user`
+
+To make infrastructure available to your users, you must add one or more VCS
+repositories containing Terraform configuration to the Terraform service
```

Comment:
```suggestion
repositories containing Terraform configurations to the Terraform service
```

Caught a couple of these so far, maybe should just make a general comment: the collective noun for an unknown number of terraform configs is "Terraform configurations," not "...configuration".